.github/workflows/ci.yml

```diff
@@ -23,6 +23,16 @@ env:
   TRAVIS_BUILD_DIR: ${{github.workspace}}
 
 jobs:
+  golangci:
```

Comment:
how does this even work? there is no `run`.

Reply:
welcome to the confusing world of github actions :sweat_smile: 

`golangci` is an arbitrary job name here but below, where it says "Run golangci-lint", there's no `:run` like in the others because it has `uses: golangci/golangci-lint-action@v1`. they are exclusive.

that tells github: go fetch from github.com/golangci/golangci-lint-action at that "version" and run their script.

Reply:
and they know it's a Go program by inspecting the source? :thinking:

Reply:
well, `golangci-lint` by definition runs only against Go code, right? :)
also, they of course use our `.golangci.yml` config which tells it what to check and not to.

Reply:
of course, if you put it against api-js repo, I doubt it'll have any useful effect :smile: